charts/camunda-platform/test/optimize/golden/deployment.golden.yaml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
:question: Looks like you copied the operate golden file. Does this actually work? I would expect everywhere it says operate to be replaced by optimize? I checked the Optimize templates/deployment, and I'm confused, since I would expect it to say optimize as well.

Reply:
If it works, then maybe the tests are not working properly?

Reply:
Ah, looks like we're testing against the Operate chart :smile:

Reply:
ups :D

Reply:
Thanks @npepinpe